src/pages/staticPages/tutorials/Tutorials.tsx

```diff
@@ -24,7 +24,7 @@ export default class Tutorials extends React.Component<{}, {}> {
                 [<a href={getNCBIlink('/pubmed/23550210')}>Reprint</a>].</p>
 
             <hr />
-            <h2>View tutorial slides below or download slides <a href="https://drive.google.com/drive/u/0/folders/0B9KTQJAGhFhhRi1qaTdUWmpLQTA">here</a>.</h2>
+            <h2>View tutorial slides below or download from <a href="https://drive.google.com/drive/u/0/folders/0B9KTQJAGhFhhRi1qaTdUWmpLQTA">Google Drive</a> or <a href="https://github.com/cBioPortal/cbioportal/tree/master/docs/tutorials">GitHub</a>.</h2>
```

Comment:
@dippindots there should be some property that one can use to get the github url

Reply:
i think this one: https://github.com/cBioPortal/cbioportal-frontend/blob/master/src/config/IAppConfig.ts#L59

Reply:
@inodb I don't see how to see the original GitHub tutorial folder by using this URL (): https://raw.githubusercontent.com/cBioPortal/cbioportal/master/docs/tutorials. I think the `raw.githubusercontent.com` is only for the purpose of download a specific file. Don't we need to access the GitHub folder by this URL: https://github.com/cBioPortal/cbioportal/tree/master/docs/tutorials?

Reply:
could do a substitution of `https://raw.githubusercontent.com/` to `https://github.com/`